### PR TITLE
Feature/572

### PR DESCRIPTION
**Pull request by MMashivskij**
Was like this.
image

Now it like this.
image

Create new component 'ImageUploadPatternContainer' (../containers/image-upload-container/image-upload-patternContainer).
Add it to 'pattern-form.js'

### CODE REVIEW

File: src/components/forms/pattern-form/pattern-form.js [hunk 1]
```diff
@@ -221,17 +220,16 @@ const PatternForm = ({ pattern, id, isEdit }) => {
...
-                  <ImageUploadContainer handler={handleLoadMainImage} />
-                  {patternImage && (
-                    <Avatar src={patternImage}>
-                      <Image />
-                    </Avatar>
-                  )}
+                  <ImageUploadPatternContainer
+                    handler={handleLoadMainImage}
+                    src={patternImage}
+                    id='patternImage'
```
Comment (AndriyLymych) [inline src/components/forms/pattern-form/pattern-form.js:231]: dont use magic strings

File: src/components/forms/pattern-form/pattern-form.js [hunk 2]
```diff
@@ -242,13 +240,13 @@ const PatternForm = ({ pattern, id, isEdit }) => {
...
-                  <ImageUploadContainer handler={handleLoadConstructorImage} />
-                  {constructorImg && (
-                    <Avatar src={constructorImg}>
-                      <Image />
-                    </Avatar>
-                  )}
+                  <ImageUploadPatternContainer
+                    handler={handleLoadConstructorImage}
+                    src={constructorImg}
+                    id='constructorImg'
```
Comment (AndriyLymych) [inline src/components/forms/pattern-form/pattern-form.js:248]: same

**Comment by Kiyooooko** [comment]
move to our Horondi board
Screenshot 2021-02-23 at 10 44 31

File: src/containers/image-upload-container/image-upload-patternContainer.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  return (
+    <div>
+      <label htmlFor={id} data-cy='pattern-image'>
+        <img className={style.image} src={src || upload} alt='pattern' />
+        <input
+          style={{ display: 'none' }}
+          id={id}
+          name='upload-photo'
+          type='file'
+          multiple
```
Comment (AndriyLymych) [inline src/containers/image-upload-container/image-upload-patternContainer.js:18]: same

**Comment by AndriyLymych** [comment]
image
remove empty file

**Comment by Kiyooooko** [comment]
move text 'Фото' and 'Фото для конструктора' over image
Screenshot 2021-02-23 at 10 45 15

**Comment by MMashivskij** [comment]
> image
> remove empty file

image
It`s not empty.

File: src/pages/home-page/home-page.js [hunk 4]
```diff
@@ -28,6 +28,7 @@ const HomePage = () => {
 
   const dispatch = useDispatch();
   const { loading, photos } = useSelector(selectPhotosAndLoading);
+  console.log(photos);
```
Comment (jul4ajka) [inline src/pages/home-page/home-page.js:31]: Delete console.log

File: src/containers/image-upload-container/image-upload-patternContainer.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+const ImageUploadPatternContainer = ({ handler, multiple, src, id }) => {
+  const style = useStyles();
+  const uploadPhoto = 'upload-photo';
+
+  return (
+    <div>
+      <label htmlFor={id} data-cy='pattern-image'>
+        <img className={style.image} src={src || upload} alt='pattern' />
+        <input
+          style={{ display: 'none' }}
```
Comment (jul4ajka) [inline src/containers/image-upload-container/image-upload-patternContainer.js:15]: Remove to the styles

File: src/containers/image-upload-container/image-upload-patternContainer.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import upload from '../../assets/images/upload.png';
+import { useStyles } from './image-upload-patternContainer.styles';
+
+const ImageUploadPatternContainer = ({ handler, multiple, src, id }) => {
+  const style = useStyles();
+  const uploadPhoto = 'upload-photo';
```
Comment (jul4ajka) [inline src/containers/image-upload-container/image-upload-patternContainer.js:8]: Move to utils

**Review by jul4ajka — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Review by Kiyooooko — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Pull request merged**